src/Text/Pandoc/Writers/Typst.hs

```diff
@@ -208,8 +208,8 @@ inlineToTypst inline =
               DisplayMath -> return $ "$ " <> literal r <> " $"
     Code (_,cls,_) code -> return $
       case cls of
-        (lang:_) -> "#raw(lang=" <> doubleQuotes (literal lang) <>
-                        ", " <> doubleQuotes (literal code) <> ")"
+        (lang:_) -> "#raw(lang:" <> doubleQuotes (literal lang) <>
+                    ", " <> doubleQuotes (literal $ T.replace "\"" "\\\"" code) <> ")"
```

Comment:
Oh right!  This makes me think we need to worry about other escapes too.
Let me check on the site.

Reply:
Yeah, we also need to backslash escape any backslashes in the code.

Reply:
I think it would be good to add a general function that handles this for everything we double quote -- I will do this, so don't worry about updating here.